.github/workflows/post-merge.yml

```diff
@@ -0,0 +1,139 @@
+---
+name: push-downstream
+on:
+    schedule:
+        - cron: '*/20 * * * *'
+jobs:
+    push-downstream:
```

Comment:
I think I'd be happier with a job-per-downstream and a `downstream-master` tag per-downstream. That way, an individual downstream can fail for whatever reason (eg: Terraform's README.md is updated and a sync fails) and the others won't be affected.

Reply:
I'd like to second that, largely so we don't have sadness where we're blocked on another repo like we were during 3.0.0-beta.1 development, needing to close spurious PRs before we could merge.